ru.js

```diff
@@ -920,63 +920,63 @@ chall.name.dmg Нанесено {0} урона {1}
-commands.god.on God mode - Enabled
-commands.god.off God mode - Disabled
-commands.god.on.other God mode - Enabled for {0}
-commands.god.off.other God mode - Disabled for {0}
-commands.smite.on Smite - Enabled
-commands.smite.off Smite - Disabled
-commands.smite.on.other Smite - Enabled for {0}
-commands.smite.off.other Smite - Disabled for {0}
-commands.perk Gave {0} {1}
-commands.maxperk Gave {0} MAX {1}
-commands.unlimited.on Unlimited Ammo - Enabled
-commands.unlimited.off Unlimited Ammo - Disabled
-commands.unlimited.on.other Unlimited Ammo - Enabled for {0}
-commands.unlimited.off.other Unlimited Ammo - Disabled for {0}
-commands.spraybypass.on Unlimited Ammo - Enabled
-commands.spraybypass.off Unlimited Ammo - Disabled
-commands.spraybypass.on.other Unlimited Ammo - Enabled for {0}
-commands.spraybypass.off.other Unlimited Ammo - Disabled for {0}
-commands.suspect.invalid Invalid suspect info
-commands.suspect.menu Suspect in menu
-commands.suspect.left Suspect has left
-commands.suspect.found Suspect Found. Spectating: {0}, Spawned: {1}, Name: {2}
-commands.forfeit.next Must play atleast 1 round
-commands.forfeit.wait Wait till match starts to forfeit
-commands.forfeit.later Please try again in a few seconds
-commands.forfeit.voted You already voted.
-commands.forfeit.limit You can only start one vote per game
-commands.forfeit.ended Forfeiting Game
-commands.forfeit.ended.other Other Team Forfeited
-server.message.join {0} joined the game
-server.message.left {0} left the game
-server.message.kicked {0} was kicked
-server.message.banned {0} was banned
-server.message.votekicked {0} was vote kicked
-server.message.arrested {0} was arrested by the K.P.D
+event.doubleXP В этом матче вы будете получать 2x больше опыта на класс
+ranked.join Вы присоединились к ранговой игре. За преждевременный выход до конца матча вы получите временный бан.
+ranked.progress Игра продолжается.
+ranked.team Пожалуйста, присоединитесь к команде
+ranked.switch Вы не можете менять класс/команду во время игры
+ranked.spectate Вы не можете играть за наблюдателя в ранговом матче
+ranked.login Войдите в аккаунт для начала
+ranked.sync Не удается синхронизировать данные
+ranked.team.spawn Присоединитесь к команде для начала
+ranked.class.spawn Выберите класс для начала
+modes.round.win Ваша команда победила этот раунд!
+modes.round.loss Ваша команда проиграла этот раунд
+modes.round.switch Смена сторон. Приготовьтесь к следующему раунду
+modes.trade.success Успешный обмен!
+modes.prophunt.limit [Prop Hunt] Вы больше не можете изменить свой облик!
+modes.prophunt.left [Prop Hunt] Вы можете изменить облик {0} раз
+modes.prophunt.rotation.on [Prop Hunt] Вращение включено
+modes.prophunt.rotation.off [Prop Hunt] Вращение выключено
+modes.prophunt.rotation.reset [Prop Hunt] Сброс вращения.
+commands.login Войдите, чтобы использовать эту команду
+commands.connected {0} подключено, {1} в меню
+commands.host {0} является хостом.
+commands.noclip.on Ноклип включен
```

Comment:
```suggestion
commands.noclip.on Noclip включен
```
Функции, у которых нет нормального перевода, не стоит транслитировать

Reply:
мда, я дурачок :/